src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,62 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.ToDo;
+import duke.util.DukeException;
+
+import java.util.Hashtable;
+import java.util.Scanner;
+import java.util.function.Consumer;
+
+public class Duke {
```

Comment:
Could you add some header comments or comments in the class ? It would explain what the purpose of the class is.